content/docs/operating/configuration.md

```diff
@@ -150,6 +154,37 @@ names:
 Where `<host>` is a valid hostname.
 
 
+### Consul SD configurations `<consul_sd_config>`
+
+Consul SD configurations allow to retrieve scrape targets from [Consul's](https://www.consul.io)
+Catalog API. 
+
+The following labels are attached to targets:
+
+* `__meta_consul_node`: the node name defined for the target
```

Comment:
All other reserved label names also end with two underscores - wondering if we should do the same here for consistency?